buildenv/jenkins/docker-slaves/README.md

```diff
@@ -0,0 +1,66 @@
+<!--
+Copyright (c) 2018, 2018 IBM Corp. and others
+ 
+ This program and the accompanying materials are made available under
+ the terms of the Eclipse Public License 2.0 which accompanies this
+ distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ or the Apache License, Version 2.0 which accompanies this distribution 
+and
+ is available at https://www.apache.org/licenses/LICENSE-2.0.
+ 
+ This Source Code may also be made available under the following
+ Secondary Licenses when the conditions for such availability set
+ forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ General Public License, version 2 with the GNU Classpath
+ Exception [1] and GNU General Public License, version 2 with the
+ OpenJDK Assembly Exception [2].
+ 
+ [1] https://www.gnu.org/software/classpath/license.html
+ [2] http://openjdk.java.net/legal/assembly-exception.html
+ 
+ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH 
+Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+# Docker
+## About
+Dockerfiles are used to define how a docker container/image
+should be built. Docker containers are essentially virtual
+machines designed for short term use. This makes them very
+useful for testing purposes as you can generate a new container
+for every test, so you will know you are running against the
+same environment every time.
+## Platforms currently available:
+- x86 Centos6.9
+- x86 Ubuntu16
+- x86 Ubuntu18
+- s390x Ubuntu16
+- s390x Ubuntu18
+- ppc64le Centos7
+- ppc64le Ubuntu16
+- ppc64le Ubuntu18
+## Generating containers at eclipse
+The dockerfiles are used by two jenkins jobs at eclipse to
+generate docker containers. The first job is run [manually](https://ci.eclipse.org/openj9/view/Build/job/Build-Jenkins-Agent-Container/)
+by supplying the architecture and os,and should be run after
+a dockerfile change has been merged. The second job is used
+to [test pull requests](https://ci.eclipse.org/openj9/view/Pull%20Requests/job/PullRequest-Build-Jenkins-Agent-Container/) involving docker files and can be
+launched by making a comment on the pull request containing
+`jenkins build docker ARCH OS` Both jobs will push the docker
+images to dockerhub under the repo `eclipse/openj9-jenkins-agent-ARCH-OS:TAG`
+If a manual build is run the tag will be the build number of
+the job, if a test build is run the tag will contain `PR`
+followed by the ID of the pull request used to build the job.
+## Using containers from a terminal
+If you have [docker installed](https://docs.docker.com/install/) then you can run
+`docker pull eclipse/openj9-jenkins-agent-ARCH-OS:TAG` to pull
+a docker image of the specified architecture and os. 
+A list of docker commands for managing/using containers can
+be found [here](https://docs.docker.com/engine/reference/commandline/docker/).
+## Docker and Jenkins
+Jenkins supports a docker plugin which allows jenkins to access docker 
```

Comment:
`There is a Docker plugin for Jenkins which allows Jenkins to orchestrate running Docker containers on remote machine(s).`